.gitignore

```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
```

Comment:
but I LOVE checking the 2D grid location from my computer into the repo :'(

Reply:
Too bad the auto-generated gitignore doesn't want you to